is_left_camera: true

acquisition_frame_rate: &frame_rate 10.0

# Trigger Settings
trigger_selector: "FrameStart"
trigger_source: "Line0"
enable_trigger: false

# Diagnostics configuration:
desired_freq: *frame_rate
